scripts/prepare-tests.sh

```diff
@@ -35,7 +35,7 @@ cat os.yml.fusor >> operatingsystems.yml
 rm -f hosts.yml os.yml.foreman os.yml.fusor
 ln -s ../../../katello/test/fixtures/models/* .
 
-# Create hosts file
-echo "" >> hosts.yml
-cat ../../../fusor/server/test/fixtures/hosts.yml >> hosts.yml
+# Re-link fusor hosts.yml file
```

Comment:
While developing, I couldn't figure out why my new hosts fixtures weren't showing up, so I ended up linking to ours instead.  It should be the same contents though since it used to blank it out then copy it into the katello/../hosts.yml